csharp/src/Ice/OutgoingResponseFrame.cs

```diff
@@ -174,6 +182,11 @@ public static OutgoingResponseFrame WithVoidReturnValue(Current current)
                         var replyStatus = (ReplyStatus)response.Payload[0];
                         if (replyStatus == ReplyStatus.UserException)
                         {
+                            // We are forwarding an ice1 user exception using ice2. We create an ice2 encapsulation
+                            // and write the 1.1 reply status followed by the user exception. The size of the new
+                            // encapsulation is adjusted: -1 corresponds to a result type that is included in the
+                            // payload but is not part of the encapsulation; +1 corresponds to a 1.1 replyStatus
```

Comment:
This is not quite correct as we did not receive a result type in the ice1/1.1 payload. Maybe just remove the -1 / +1 below and remove this comment?